include/platform/mir/input/event_builder.h

```diff
@@ -38,21 +39,34 @@ class EventBuilder
 public:
     EventBuilder() = default;
     virtual ~EventBuilder() = default;
-    /// Timestamps in returned events are automatically calibrated to the Mir clock
+    /// Timestamps in returned events are automatically calibrated to the Mir clock. If nullopt is given current mir
+    /// clock time is used.
     using Timestamp = std::chrono::nanoseconds;
```

Comment:
I don't think this documentation belongs to this type.

This `Timestamp` type represent a timestamp (implemented as a duration from an implicit epoch). 

1. "calibration" is changing to a different epoch, but isn't done by `Timestamp`; and,
2. a `Timestamp` cannot be set to `nullopt`.